### PR TITLE
Admin edit films

### PR DESCRIPTION
**Pull request by carolynyoo**
admin

Can delete films and edit films!!!!! (edit gif to come)

### CODE REVIEW

File: browser/js/admin/directives/admin-sidebar/admin-sidebar.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="col-sm-3">
+  <div class="well">
+    <ul class="nav nav-list">
+      <li class="nav-header">Menu</li>
+      <li ng-repeat="item in items">
+        <a ui-sref-active="active" ui-sref="{{item.state}}">{{item.label}}</a>
```
Comment (itsaliu90) [inline browser/js/admin/directives/admin-sidebar/admin-sidebar.html:6]: That's cool

File: browser/js/admin/templates/categories.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<div class="table-responsive">
+  <table class="table table-striped">
+    <thead>
+      <tr>
+      <th></th>
+      <th>Categories</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr ng-repeat="category in categories | orderBy:'name'">
```
Comment (itsaliu90) [inline browser/js/admin/templates/categories.html:11]: Woo filters!

File: browser/js/admin/templates/welcome.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Welcome to B.A. Movies Admin, {{user.name}}!</h1>
+<p>CRUD to your heart's desire. </p>
```
Comment (itsaliu90) [inline browser/js/admin/templates/welcome.html:2]: :ghost:

File: seed.js [hunk 4]
```diff
@@ -136,6 +136,30 @@ var seedFilms = [
...
+    {
+        title: 'Singin\' in the Rain',
+        categories: ['Musical'],
+        description: 'A spoof of the turmoil that afflicted the movie industry in the late 1920s when movies went from silent to sound. ',
+        price: 199,
+        photo: 'http://image.tmdb.org/t/p/original/bLbZBIeEqBcDiqUmLare2nv8fAt.jpg',
+        inventory: 2
+    },
+    {
+        title: 'Memento',
```
Comment (itsaliu90) [inline seed.js:157]: Great movie!

**Comment by itsaliu90** [comment]
:hamburger:

**Comment by darksim735** [comment]
:hamburger:

**Pull request merged**